pkg/util/tests/kubectl/kubectl.go

```diff
@@ -0,0 +1,291 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR Condition OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubectl
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/istio-ecosystem/sail-operator/pkg/util/tests/shell"
+	resourcecondition "github.com/istio-ecosystem/sail-operator/pkg/util/tests/types"
+)
+
+const DefaultCommandTool = "kubectl"
+
+var (
+	ErrNotFound       = errors.New("resource was not found")
+	EmptyResourceList = resourcecondition.ResourceList{
```

Comment:
I'll use dot import to import directly the type

Reply:
Hmm, not sure about that. We should keep dot imports to a minimum. It makes sense for ginkgo/gomega, since that's a DSL, but for other stuff, I think we should just use proper imports.